src/pages/gettingStarted.ts

```diff
@@ -0,0 +1,121 @@
+export const GETTING_STARTED_HTML = `<!DOCTYPE html>
+    <html>
+        <head>
+            <title></title>
+        </head>
+        <style>
+            .inv {
+                display: none;
+            }
+            .deviceSelector {
+                width: 250px;
+                border: 1px solid #3D484C;
+                margin: 0 0 5px;
+                padding: 8px;
+                border-radius: 5px;
+                font-size: 12px;
+                padding-right: 30px;
+            }
+            .codeBox {
+                display: block;
+                width: 90%;
+                margin: 10px;
+                padding: 15px;
+                text-align: left;
+                background: none;
+                border: 1px solid grey;
+                border-radius:4px;
+            }
+
+        </style>
+        <body>
+            <h1>Getting started</h1>
+
+            <select id="target" class="deviceSelector">
+                <option selected disabled>Select a Device</option>
+                <option value="micro:bit">micro:bit</option>
+                <option value="CPX">CPX</option>
+            <select>
```

Comment:
When you first open the getting started page, `Select a Device` is selected but the micro:bit tutorial is showed.

Reply:
I've put microbit as default device. We can think of a better flow.